recipes/libs/zstd.yaml

```
@@ -0,0 +1,35 @@
+inherit: [cpackage, make, install, patch, strip]
+
+metaEnvironment:
+  PKG_VERSION: "1.5.6"
+
+checkoutSCM:
+  scm: url
+  url: ${GITHUB_MIRROR}/facebook/zstd/releases/download/v${PKG_VERSION}/zstd-${PKG_VERSION}.tar.gz
+  stripComponents: 1
+
+checkoutScript: |
+  patchApplySeries $<@zstd/*@>
+
+buildVars: [CROSS_COMPILE, PKG_VERSION]
+buildScript: |
+  export CC=${CROSS_COMPILE}gcc
+  export CXX=${CROSS_COMPILE}g++
+  export AR=${CROSS_COMPILE}ar
+
+  mkdir -p install build
+  cp -a $1/* build # out of tree building does not work, zstd writes to source tree
+  export DESTDIR=${PWD}/install # zstd Makefile variable
+
+  export LIB_SRCDIR=$PWD/build/lib
+
+  pushd build
+  makeParallel -C $LIB_SRCDIR PREFIX=/usr
+  makeParallel -C $LIB_SRCDIR/../programs PREFIX=/usr
+  makeParallel -C $LIB_SRCDIR install PREFIX=/usr
+  popd
+
+packageScript: |
+  cp -a $1/install/* .
+  stripAll .
```

As this is a library, it should probably have a `dev` and `tgt` package. Something along these lines:

```suggestion
multiPackage:
  dev:
    packageScript: |
      installPackageDev "$1/install/"
  tgt:
    packageScript: |
      installPackageTgt "$1/install/"
```